_posts/2018-12-08-bit-arpa-feedback.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Seeking feedback on .bit.arpa as special-use domain name"
+author: Jeremy Rand
+tags: [News]
+---
+
+We're seeking feedback on a proposal to use `.bit.arpa` as a special-use domain name for Namecoin.  [Our current evaluation of the proposal is here.]({{site.baseurl}}docs/dev/bit-arpa/)  We won't be doing anything in this department imminently, but we'd greatly appreciate community feedback -- please take some time to read our evaluation and let us know what you think.
```

Comment:
Immediately..?

Reply:
More-correct to be "immediately" as per colloquial usage; "without intervening time period." There is an intervening time period, so action does not take place immediately. Imminence implies the action is, to some implied degree, inevitable. Imminence also has an amount of other baggage w.r.t natural disasters and so on. Not as apt.